app/views/people/_form.html.erb

```diff
@@ -48,6 +48,9 @@
     <%= f.input :start_date, :as => :string, :input_html => { :class => 'datepicker', :size => 11 }  %>
     <%= f.input :division1, :as => :select, :label => 'Division', :collection => Person::DIVISION1 %>
     <%= f.input :division2, :as => :select, :label => 'Squad', :collection => Person::DIVISION2 %>
+    <h4> Personnel Image </h4>
+    <%= @person.portrait %>
+    <%= f.file_field :portrait %>
```

Comment:
For some reason I have a broken "Browse..."  button displayed.  It is missing the bottom line. See pictures above and below.